lib/domain/timeline/timeline_icon.dart

```diff
@@ -1,314 +1,314 @@
 enum TimelineIcon {
-  NOTIFICATION_REMINDER,
-  HOCKEY_GAME,
-  PAY_BILL,
-  NOTIFICATION_LINKEDIN,
-  NOTIFICATION_GOOGLE_INBOX,
-  GENERIC_QUESTION,
-  NOTIFICATION_FLAG,
-  GENERIC_SMS,
-  WATCH_DISCONNECTED,
-  TV_SHOW,
-  BASKETBALL,
-  GENERIC_WARNING,
-  LIGHT_RAIN,
-  NOTIFICATION_FACEBOOK,
-  INCOMING_PHONE_CALL,
-  NOTIFICATION_GOOGLE_MESSENGER,
-  NOTIFICATION_TELEGRAM,
-  NOTIFICATION_FACETIME,
-  ARROW_DOWN,
-  NOTIFICATION_OUTLOOK,
-  NO_EVENTS,
-  AUDIO_CASSETTE,
-  SUNSET,
-  NOTIFICATION_TWITTER,
-  SUNRISE,
-  HEAVY_RAIN,
-  NOTIFICATION_MAILBOX,
-  AMERICAN_FOOTBALL,
-  CAR_RENTAL,
-  CRICKET_GAME,
-  NOTIFICATION_WECHAT,
-  NOTIFICATION_GENERIC,
-  NOTIFICATION_SKYPE,
-  CLOUDY_DAY,
-  DURING_PHONE_CALL_CENTERED,
-  NOTIFICATION_LINE,
-  HOTEL_RESERVATION,
-  NOTIFICATION_FACEBOOK_MESSENGER,
-  NOTIFICATION_LIGHTHOUSE,
-  TIMELINE_EMPTY_CALENDAR,
-  NOTIFICATION_IOS_PHOTOS,
-  RESULT_DELETED,
-  NOTIFICATION_GMAIL,
-  TIMELINE_MISSED_CALL,
-  SLEEP,
-  RESULT_MUTE,
-  NOTIFICATION_AMAZON,
-  THUMBS_UP,
-  SCHEDULED_FLIGHT,
-  SETTINGS,
-  PARTLY_CLOUDY,
-  STOCKS_EVENT,
-  NOTIFICATION_GOOGLE_MAPS,
-  REWARD_GOOD,
-  NOTIFICATION_YAHOO_MAIL,
-  BIRTHDAY_EVENT,
-  GENERIC_EMAIL,
-  RESULT_DISMISSED,
-  NOTIFICATION_GOOGLE_PHOTOS,
-  TIDE_IS_HIGH,
-  NOTIFICATION_VIBER,
-  LIGHT_SNOW,
-  NEWS_EVENT,
-  GENERIC_CONFIRMATION,
-  TIMELINE_SPORTS,
-  NOTIFICATION_SLACK,
-  CHECK_INTERNET_CONNECTION,
-  ACTIVITY,
-  NOTIFICATION_HIPCHAT,
-  NOTIFICATION_INSTAGRAM,
-  TIMELINE_BASEBALL,
-  REWARD_BAD,
-  REACHED_FITNESS_GOAL,
-  DAY_SEPARATOR,
-  TIMELINE_CALENDAR,
-  RAINING_AND_SNOWING,
-  RADIO_SHOW,
-  DISMISSED_PHONE_CALL,
-  ARROW_UP,
-  REWARD_AVERAGE,
-  MUSIC_EVENT,
-  NOTIFICATION_SNAPCHAT,
-  NOTIFICATION_BLACKBERRY_MESSENGER,
-  NOTIFICATION_WHATSAPP,
-  LOCATION,
-  SOCCER_GAME,
-  RESULT_FAILED,
-  RESULT_UNMUTE,
-  SCHEDULED_EVENT,
-  TIMELINE_WEATHER,
-  TIMELINE_SUN,
-  NOTIFICATION_GOOGLE_HANGOUTS,
-  DURING_PHONE_CALL,
-  NOTIFICATION_KIK,
-  RESULT_UNMUTE_ALT,
-  MOVIE_EVENT,
-  GLUCOSE_MONITOR,
-  RESULT_SENT,
-  ALARM_CLOCK,
-  HEAVY_SNOW,
-  DINNER_RESERVATION,
-  NOTIFICATION_KAKAOTALK,
+  notificationReminder,
```

Comment:
ditto TimelinePinLayout

Reply:
These numbers must be constant since they correspond to integers that get sent to the watch. I don't think generating them is a good idea since generator can change values.